app/src/main/java/com/android/brewr/ui/userProfile/UserPrivateList.kt

```diff
@@ -102,7 +71,7 @@ fun UserPrivateListScreen(
                 Modifier.fillMaxSize().testTag("privateList").padding(16.dp).padding(paddingValues),
             verticalArrangement = Arrangement.spacedBy(8.dp)) {
               if (!showPrivateCoffeeInfos) {
-                CoffeeList(listOf(mockCoffee)) {
+                CoffeeList(privateList) {
```

Comment:
Since CoffeeList function shows "Everywhere is closed! I guess Coffee will disturb your sleep at this time" message when the list is empty, the user private list is also showing the same message when there is no saved list. I think we can solve this problem by changing the message into something like "No Coffeeshop Found". How do you think about it? :)

Reply:
Good idea!